test/parallel/test-eslint-prefer-primordials.js

```diff
@@ -77,6 +77,10 @@ new RuleTester({
         `,
         options: [{ name: 'Reflect' }],
       },
+      {
+        code: 'const { Map } = primordials; new Map()',
+        options: [{ name: 'Map', into: 'Safe' }],
```

Comment:
If we prefer SafeMap to Map, all Map should be replaced with SafeMap.
So this case should be failed with the error `const { SafeMap } = primordials`.

Reply:
Fair enough, I think in the long run we might want to remove `primordials.Map` entirely. But it's not the purpose of this PR, I'm trying to nudge collaborators into using the safe alternative.
If you think it's not acceptable and are still -1 on this, I'll close this PR, I don't want to work on this for the time being.

Reply:
We need `primordials.Map` for public APIs that return `Map` objects.

Reply:
> We need `primordials.Map` for public APIs that return `Map` objects.

Or maybe we want `globalThis.Map` for those? But that's a discussion for another time.

Reply:
@aduh95 Okay, I'll work on that after this is landed.
Anyway, this PR is useful in some cases. Let's land this.